pkg/debug/Dockerfile

```diff
@@ -50,6 +51,13 @@ ADD https://github.com/pixel/hexedit/archive/refs/tags/1.5.tar.gz ../1.5.tar.gz
 RUN tar -C .. -xzvf ../1.5.tar.gz
 RUN ./autogen.sh && ./configure && make DESTDIR=/out install
 
+# building picocom
+# Need this patch to build with musl: https://github.com/npat-efault/picocom/commit/1acf1ddabaf3576b4023c4f6f09c5a3e4b086fb8
+ENV PICOCOM_COMMIT=1acf1ddabaf3576b4023c4f6f09c5a3e4b086fb8
+ADD --keep-git-dir=true https://github.com/npat-efault/picocom.git#${PICOCOM_COMMIT} /tmp/picocom
```

Comment:
we usually use the 1.5.0-rc2-labs version: `# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs`, maybe is that why hadolint catch a invalid flag?

Reply:
Changed it to `1.5.0-rc2-labs`, let's see what hadolint thinks about it...

Reply:
hadolint probably does not know `--keep-git-dir` flag

Reply:
Correct it doesnt know about that flag